test/linters/params/rustfmt.js

```diff
@@ -0,0 +1,72 @@
+const { join } = require("path");
+
+const RustFmt = require("../../../src/linters/rustfmt");
+
+const testName = "rustfmt";
+const linter = RustFmt;
+const commandPrefix = "";
+const args = " -- --color=never ";
```

Comment:
```suggestion
const args = "";
```

Let's remove it from the test file to verify that the default arg is now used.

Reply:
It may be my javascript ignorance, but it seems that doing:
```js
	static lint(dir, extensions, args = "-- --color=never", fix = false, prefix = "") {
```
doesn't work since the tests and index.js pass in `args` explicitly:
- https://github.com/wearerequired/lint-action/blob/master/test/linters/linters.test.js#L89
- https://github.com/wearerequired/lint-action/blob/master/src/index.js#L94

I did try doing:
```js
			let garbageArgs;
			test(`${linter.name} returns expected ${lintMode} output`, () => {
				const cmdOutput = linter.lint(projectTmpDir, extensions, garbageArgs, autoFix, commandPrefix);
```
which seems to fallback to the default in the method signature, but it didn't feel like a good solution. 
Again I'm not a JS dev so maybe this is a common idiom.

Reply:
Yeah, nevermind. I thought the tests were using the data from the yaml file but they don't.